client/src/pages/Main.js

```diff
@@ -122,107 +122,37 @@ class Main extends Component {
-          title: "PalPay invested $500 million in Company ABC",
-          platform: "Reddit",
-          content:
-            "Heat oil in a (14- to 16-inch) paella pan or a large, deep" +
-            "skillet over medium-high heat. Add chicken, shrimp and chorizo," +
-            "and cook, stirring occasionally until lightly browned, 6 to 8" +
-            "minutes. Transfer shrimp to a large plate and set aside, leaving" +
-            "chicken and chorizo in the pan. Add pimentón, bay leaves," +
-            "garlic, tomatoes, onion, salt and pepper, and cook, stirring" +
-            "often until thickened and fragrant, about 10 minutes. Add" +
-            "saffron broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil. broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil.",
-          image: "/imgs/dog.jpg",
-        },
-        {
-          title: "PalPay invested $500 million in Company ABC",
-          platform: "Reddit",
-          content:
-            "Heat oil in a (14- to 16-inch) paella pan or a large, deep" +
-            "skillet over medium-high heat. Add chicken, shrimp and chorizo," +
-            "and cook, stirring occasionally until lightly browned, 6 to 8" +
-            "minutes. Transfer shrimp to a large plate and set aside, leaving" +
-            "chicken and chorizo in the pan. Add pimentón, bay leaves," +
-            "garlic, tomatoes, onion, salt and pepper, and cook, stirring" +
-            "often until thickened and fragrant, about 10 minutes. Add" +
-            "saffron broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil. broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil.",
-          image: "/imgs/dog.jpg",
-        },
-        {
-          title: "PalPay invested $500 million in Company ABC",
-          platform: "Reddit",
-          content:
-            "Heat oil in a (14- to 16-inch) paella pan or a large, deep" +
-            "skillet over medium-high heat. Add chicken, shrimp and chorizo," +
-            "and cook, stirring occasionally until lightly browned, 6 to 8" +
-            "minutes. Transfer shrimp to a large plate and set aside, leaving" +
-            "chicken and chorizo in the pan. Add pimentón, bay leaves," +
-            "garlic, tomatoes, onion, salt and pepper, and cook, stirring" +
-            "often until thickened and fragrant, about 10 minutes. Add" +
-            "saffron broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil. broth and remaining 4 1/2 cups chicken broth; bring to a" +
-            "boil.",
-          image: "/imgs/dog.jpg",
-        },
-      ],
+      mentions: [],
     };
   }
 
+  sortByPopularity(mentions) {
+    mentions.sort((a, b) => {
+      return b.popularity - a.popularity
+    })
+    return mentions
+  }
+
+  sortByDate(mentions) {
+    mentions.sort((a, b) => {
+      return new Date(b.date) - new Date(a.date)
```

Comment:
The `date` field of mentions is in unix format, aka, the millisecs that have pasted from 1970 to now, so I think you can just compare them directly without using `new Date()`.
I didn't try it, correct me if I am wrong

Reply:
Ahh I see. I used postico to hardcode the date which isn't unix format. I will change this